backend/app/core/wiki_config.py

```
@@ -0,0 +1,56 @@
+        "/api/internal/wiki/generations/contents"  # Fixed relative path for content writes
+    )
+    DEFAULT_SECTION_TYPES: list[str] = [
+        "overview",
+        "architecture",
+        "module",
+        "api",
+        "guide",
+        "deep",
+    ]
+    INTERNAL_API_TOKEN: str = (
+        "weki"  # Internal authentication token for content write API
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "INTERNAL_API_TOKEN"

(S105)